cython_vagrant/salt/roots/salt/cython_sample.sls

```diff
@@ -0,0 +1,5 @@
+https://github.com/clsung/cythonup.git:
```

Comment:
@clsung You haven't accepted PR #1 so that no once can change it in the blessed repo.